### ISSUE TITLE
Serial Steps

### ISSUE THREAD

**Issue by srushti** [srushti/goldberg#66]
I should also be able to define steps in the build where one step (or multiple if #65 has already been implemented) has to be run before one (or multiple) other steps. I should be able to specify different configurations for each step.

**Comment by praveenram** [comment]
I was looking to implement pipeline stages in the CI and found its still work in progress. Is there any ETA or details on how much work is complete?

There is an alternate way to have basic pipeline stages however at this point in time using the project success handler.

**Comment by srushti** [comment]
No, there hasn't been anything implemented on this. These were part of larger plans for Goldberg which would support pipelines and the like. I'm not sure when I'll actually be getting to them.

**Comment by praveenram** [comment]
Thanks, will look at it.
